### DISCUSSION TITLE
medical not allowed

### DISCUSSION THREAD

**Discussion by tolot27** [tolot27/xDrip#34] [General]
Is it allowed to discuss about medical usage of xDrip?

**Comment by MasterPlexus** [comment]
It's an good question. Officially the app ist not marked as ce medical device. But I think this could be handled with a disclaimer to make it clear (lang time ago that I started xDrip the first time, but I think such a remark was included at the first start Screen?) But what is meant with not allowed to discuss the use as a medical device? This for me is not clear.

**Reply by tolot27** [reply]
> But what is meant with not allowed to discuss the use as a medical device? This for me is not clear.

I just created an example and my post is based on Jon's concern at Gitter: "There is also that xDrip is not for medical use and so discussions would need to avoid anything that comes close to medical advice."

I did not really wan't to enter this discussion.

**Comment by tolot27** [comment]
This discussion is just for testing and has no official character and meaning. Just to try things out and help in deciding whether it could be a substitute of or addition for the Gitter/Facebook channels.

**Comment by MasterPlexus** [comment]
So then I could say anything, all and fourtytwo ;) and I will do.

But to check if this way of discussion will work more should try then...

Mathias Walter <notifications@github.com> schrieb am Do., 18. Feb. 2021,
23:16:

**Reply by tolot27** [reply]
Yes, but this should be done at the original repository and not at my fork. It can be announced that this is a test period and if the main developers and main supporters agree with how it evolves, it will be kept enabled. Otherwise, it will be stopped.
